src/components/GoogleDrive.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import Radium from "radium";
+
+
+//TODO
+//so much
+//research google APIs
+@Radium
+export default class Connect2GDriveButton extends React.Component {
+  constructor(props){
+    super(props);
+  }
+  render() {
+    return(
+      <button style={[]}>Connect to Google Drive</button>
+    );
+  }
+}
```

Comment:
Let's leave this component for later. Move the code you have here into a separate branch for a separate future pull request.